internal/controller/impersonatorconfig/impersonator_config.go

```diff
@@ -313,6 +312,23 @@ func (c *impersonatorConfigController) doSync(syncCtx controllerlib.Context, cre
 	return credentialIssuerStrategyResult, nil
 }
 
+func (c *impersonatorConfigController) ensureCAAndTLSSecrets(ctx context.Context, nameInfo *certNameInfo) (impersonationCABundle []byte, err error) {
```

Comment:
I don't think you want to name the return types here, do you?

Reply:
I'll remove with the next push